src/types/statusBox.ts

```
@@ -5,7 +5,12 @@ export interface StatusDefaultProps {
   message: String;
   textColor?: String;
 }
+export interface StatusBaytteryAlertProps {
+  icon: IconName;
+  title: String;
+  description: String;
+}
 
 export interface StatusIndicatorProps extends StatusDefaultProps {
   bgColor: String;
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)